src/main/java/frc/robot/subsystems/grabber/GrabberIO.java

```diff
@@ -0,0 +1,17 @@
+package frc.robot.subsystems.grabber;
+
+public interface GrabberIO {
+    default void updateInputs(GrabberInputs inputs) {
+    }
+
+    default void stop() {
+        setTargetVoltage(0.0);
+    }
+
+    default void setTargetVoltage(double voltage) {
+    }
+
+    default boolean getBeamBreakState() {
```

Comment:
Remove getter in IO interface. Instead of calling a getter, the `GrabberInputs` should be used to read the value.